src/unix/notbsd/linux/mod.rs

```diff
@@ -83,9 +83,11 @@ s! {
     }
 
     pub struct pthread_mutexattr_t {
-        #[cfg(target_arch = "x86_64")]
+        #[cfg(any(target_arch = "x86_64", target_arch = "powerpc64",
+                  target_arch = "powerpc64le"))]
```

Comment:
Could you change this to `target_pointer_width`? Looks like that may be what's in play here